test/units/querySelector-dom-type.ts

```diff
@@ -0,0 +1,22 @@
+///<reference lib="dom" />
```

Comment:
Can you remove this? As much as I'd like to, we don't currently fully adhere to the TypeScript DOM typings